internal/backends/backend.go

```diff
@@ -134,6 +136,24 @@ func (bc *backendContract) DropDatabase(ctx context.Context, params *DropDatabas
 	return err
 }
 
+// ServerStatsParams represents the parameters of Backend.Stats method.
+type ServerStatsParams struct{}
+
+// ServerStatsResult represents the results of Backend.Stats method.
+type ServerStatsResult struct {
+	CountCollections int64
+}
+
+// Stats returns statistics about the server.
+func (bc *backendContract) Stats(ctx context.Context, params *ServerStatsParams) (*ServerStatsResult, error) {
```

Comment:
Ok, I think I'm getting there :) So we have `Stats` for collections, databases, and server. `Stats` for collection and database are pretty similar (e.g. fields are similar with the same meaning), but stats for server is a totally different thing... Should it be called `Status` then? Or is it used in `msg_<something>stats` as well and this is why it's called `Stats`?

Reply:
Let's use serverStatus, update the name.